src/app/core/filter/filters/filters.ts

```diff
@@ -153,113 +83,37 @@ export class SelectableFilter<T extends Entity> extends Filter<T> {
     public label: string = name,
   ) {
     super(name, label);
-    this.selectedOption = this.options[0]?.key;
+    this.selectedOptionsKeys = this.options.map(
+      (value: FilterSelectionOption<T>) => value.key,
+    );
```

Comment:
Any reason why all oh them need to be assigned initially?

Reply:
No. Done.